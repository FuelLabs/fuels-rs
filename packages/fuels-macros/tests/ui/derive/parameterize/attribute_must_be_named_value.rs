use fuels_macros::Parameterize;

#[derive(Parameterize)]
#[FuelsTypesPath]
enum SomeEnum {
    A(u8),
}

#[derive(Parameterize)]
#[FuelsTypesPath = true]
struct SomeStruct {}

fn main() {}
