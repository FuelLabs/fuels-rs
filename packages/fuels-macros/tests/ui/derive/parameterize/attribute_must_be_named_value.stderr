error: expected name='value'
 --> tests/ui/derive/parameterize/attribute_must_be_named_value.rs:4:3
  |
4 | #[FuelsTypesPath]
  |   ^^^^^^^^^^^^^^

error: expected string literal
  --> tests/ui/derive/parameterize/attribute_must_be_named_value.rs:10:20
   |
10 | #[FuelsTypesPath = true]
   |                    ^^^^
